R/class-RDAPlot.R

```diff
@@ -260,6 +301,8 @@ setMethod(".definePanelTour", "RDAPlot", function(x) {
         .selectInput.iSEE(x, field="add.ellipse", label="Ellipse style",
             choices = c("fill", "colour", "FALSE"),
             selected=slot(x, "add.ellipse")),
+        .sliderInput.iSEE(x, field="ellipse.alpha", label="Ellipse opacity",
+            min=0, max=1, step=10, value=slot(x, "ellipse.alpha")),
```

Comment:
try reducing the step size for example to 0.1. I think this param is also missing in the .createObservers method